### PR TITLE
Feat/display images

### LINKED ISSUE #12: Make `Filter Recipes` search very clear with what is found
Problem: `Filter Recipes` search does not always make sense
- `Hor D'oeuvre` is not returning the found recipes for any reason clear to the user
- all recipe types should be clear
Desired Result: `Filter Recipes` search should make sense because recipes display the relevant recipe type

### PR DESCRIPTION
**Pull request by rachelbuchta**
**What does this PR do?**

* imported images into scripts file so that they appear on the site
* Added a helper method to create html elements for each tag in every recipe and display it on the recipes card so that the filter by tag check boxes rendered the correct recipes
* Added new styling to the css for recipe cards to be set up with a grid in order to display the tags in a nice way.

**What does it fix?**

* Fixes the bug that didn't filter recipes correctly since not all of the tags for each recipe were showing up

**Is this a feature or a fix?**

* Fix

**Where should the reviewer start?**

* Line 75 in scripts.js

**How should this be tested?**

* Open up the browser and attempt checking off boxes for whatever tag of recipes you want to see

**Do you have any questions or issues about this addition?**

* Will definitely still need to do some. refactoring but it works!

- [x] Did you test your code locally first?
- [x] Does this follow our intended guidelines (DRY & commented) for project code?

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -60,27 +68,48 @@ function createCards() {
       shortRecipeName = recipeInfo.name.substring(0, 40) + "...";
     }
     addToDom(recipeInfo, shortRecipeName)
+    //change that shit^
   });
+ 
+}
+
+function getTags(tags) {
+  let result = ''
```
Comment (mainlyetcetera) [inline src/scripts.js]: I'm being a bit nit-picky here. Please put in the semi-colons at the end of these lines! :)

File: src/scripts.js [hunk 2]
```diff
@@ -60,27 +68,48 @@ function createCards() {
...
+  tags.forEach(tag => {
+   let newTag = `<h4>${tag}</h4>`
```
Comment (mainlyetcetera) [inline src/scripts.js]: Lint picked up something small. Please give these lines 4 spaces of indentation instead of 3. Same goes for line 80.

File: src/scripts.js [hunk 3]
```diff
@@ -60,27 +68,48 @@ function createCards() {
...
+ 
+}
+
+function getTags(tags) {
+  let result = ''
+  tags.forEach(tag => {
+   let newTag = `<h4>${tag}</h4>` 
+   result += newTag
+  })
+return result
```
Comment (mainlyetcetera) [inline src/scripts.js]: 2 spaces of indentation here please! :)

Our old style guide for mod 1 wants a space between any multi-line block and another statement, but since the lint doesn't care, neither do I.

File: src/scripts.js [hunk 4]
```diff
@@ -60,27 +68,48 @@ function createCards() {
...
 }
 
-function addToDom(recipeInfo, shortRecipeName) {
+function addToDom(recipe, shortRecipeName) {
```
Comment (mainlyetcetera) [inline src/scripts.js]: `shortRecipeName` is no longer used after your refactor. Would you go ahead and remove it as a parameter entirely please? :)

**Review by mainlyetcetera — CHANGES_REQUESTED** [review]
Logic is good. Page is functional.

I'm asking for a couple changes which are all style. Semi-colons, mostly, and the linter would like more uniform indentation. I have comments on the individual lines. 

Also, looks like we're no longer using on the of the parameters on line 85. Let's take it out please!

Overall great job putting this together! Once the changes are committed, I'll go ahead and merge this!

**Comment by mainlyetcetera** [comment]
Rachel, also found one small error with the DOM. Looks like if we've got 7 or more tags, there's not quite enough space on the card rn for the apple.
[img](https://user-images.githubusercontent.com/70294115/103956157-2907de80-5105-11eb-8145-96bf2a3c0def.png)

We don't need to fix this right now; we can make a ticket and address it later if you want.

File: src/index.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <header>
-      <h1><span>fresh</span>Picks<img src="../images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
+      <h1><span>fresh</span>Picks<img src="./images/apple-logo.png" alt="apple icon" class="header-apple-icon"></h1>
```
Comment (mattdeann) [inline src/index.html:13]: figures

File: src/scripts.js [hunk 6]
```diff
@@ -5,6 +5,12 @@ import ingredientData from './data/ingredient-data';
 
 import './css/base.scss';
 import './css/styles.scss';
+import './images/apple-logo.png'
```
Comment (mattdeann) [inline src/scripts.js:8]: Nice catch! This is via webpack import right?

File: src/css/styles.scss [hunk 7]
```diff
@@ -247,6 +248,13 @@ main {
   width: 250px;
 }
 
+.tags {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr;
+  width: 60%;
+}
+
```
Comment (mattdeann) [inline src/css/styles.scss]: I think even with the two columns, overflow scroll might still be a nice fix

File: package-lock.json [hunk 8]
```diff
@@ -1,11625 +1,8 @@
...
-    },
-    "node_modules/path-dirname": {
-      "version": "1.0.2",
-      "resolved": "https://registry.npmjs.org/path-dirname/-/path-dirname-1.0.2.tgz",
-      "integrity": "sha1-zDPSTVJeCZpTiMAzbG4yuRYGCeA=",
-      "dev": true
-    },
-    "node_modules/path-exists": {
-      "version": "4.0.0",
-      "r
```
Comment (mattdeann) [inline package-lock.json:11622]: Why is this deleted? (this sounds aggressive but I mean it curiously)

File: src/css/styles.scss [hunk 9]
```diff
@@ -236,7 +238,22 @@ main {
   flex-wrap: wrap;
 }
 
+.title-container {
```
Comment (mattdeann) [inline src/css/styles.scss:241]: Just checking so I understand, title-container existed but there was no styling for it when we got the project, right?

**Review by mattdeann — COMMENTED** [review]
Just little things from me!

File: src/scripts.js (hunk 6 above)
Comment (rachelbuchta) [reply src/scripts.js:8]: yup, i think so! In order to have access to those image you have to import them

File: src/css/styles.scss (hunk 7 above)
Comment (rachelbuchta) [reply src/css/styles.scss]: I agree!

File: package-lock.json (hunk 8 above)
Comment (rachelbuchta) [reply package-lock.json:11622]: honestly......I have no idea why that is deleted or how it even got deleted? Im confused on this one. We will have too look at this

File: src/css/styles.scss (hunk 9 above)
Comment (rachelbuchta) [reply src/css/styles.scss:241]: Nope, it didn't exist at all. I added it in the recipe cards so that I could style it with grid and have align and go where I wanted them to go when I added all of the other tags
